vtex/toolbelt
# Feature/billing options terms

#### What is the purpose of this pull request?


## What problem is this solving?


### Some context about what already exists

VTEX accounts can install apps using our CLI/toolbelt by using the command `vtex install`. Some of those apps have `billingOptions` (an optional field in the `manifest.json` file that describes how/if the app is paid). Apps without `billingOptions` are not relevant to this PR.

An app with `billingOptions` can be free or paid. In both cases, when the user tries to install (by using `$ vtex install`) an app, she is informed by the toolbelt that the app is free or paid. Also, all of these apps can optionally have a "Terms of Service" URL, which the user can visit to see the app's terms.

**The case of paid apps**

If the app is paid, it's shown to the user how exactly she is going to be possibly charged by the app. There are only two ways the user can be charged for using an app:

1) A monthly subscription price in reais (BRL, R$), like Netflix and Spotify.
2) A variable ammount according to some metric value. For example, the app `sms-provider` charges `R$ 0.07` for each SMS sent by the account that installed the app. If the account sent no SMSs, the account will not be charged at all. If it sent 10 SMS, it will be charged `R$ 0.70` at the end of the month.

Note that can 1) or 2) can both exist simultanously. 

### Who is the user or who are we talking to?

The user is anyone logged in a VTEX account and is interested in installing free or paid apps via the CLI/toolbelt.

### What does this PR changes?

1) Deal with the new `billingOptions` format (doens't impact the UI)
2) Words and colors that is shown to that is trying to install an app. No new information was added, we're just trying to make it cleaner  (impacts UI)

You can check how the UI is now by using the commands `vtex install vtex.sms-provider` or `vtex install yourviews.yourviewsreviews` (omitted here because there are a lot of pictures already, but hit me up on Slack if u need pictures).



#### How should this be manually tested?

1)  Free app without license file

```sh
vtex-test install yourviews.yourviewsreviews
```

2) Free app with license file (the URL https://extensions.myvtex.com/_v/terms/vtex.google-customer-reviews@x should work)
```sh
vtex-test install vtex.google-customer-reviews
```

3) Paid app with new `billingOptions` and license files:

```sh
vtex-test install vtex.google-shopping
```

4) One metric 

```sh
vtex-test install vtex.sms-provider
```

5) Should show subscription price and multiple metrics:

```
vtex-test install pilateslovers.billingoptionstest@0.x
```

#### Screenshots or example usage

1) 
![image](https://user-images.githubusercontent.com/12221113/94725081-9c919700-0331-11eb-921a-3bfb1bdd36bb.png)

2)
![image](https://user-images.githubusercontent.com/12221113/94725152-b3d08480-0331-11eb-811b-2ea0d58c9312.png)

3)
![image](https://user-images.githubusercontent.com/12221113/94725196-c64abe00-0331-11eb-9746-fc48dccb33d2.png)

4)
![image](https://user-images.githubusercontent.com/12221113/94725231-d2368000-0331-11eb-8a0d-8cb61520bf0e.png)

5)
![image](https://user-images.githubusercontent.com/12221113/94725865-ceefc400-0332-11eb-9c25-8ad85666ab60.png)


#### Types of changes

#### Chores checklist
